docker/docker-compose.yml.example

```diff
@@ -37,3 +9,44 @@ services:
     # MongoDB crash with permission denied errors if the command is not overridden like this
     command: bash -c 'chown -R mongodb:mongodb /data && chmod -R 755 /data && mongod'
     restart: always
+
+  postgres:
+    image: postgres:latest
+    container_name: postgres
+    ports:
+      - "5432:5432"
+    environment:
+      POSTGRES_PASSWORD: qwerty
+      POSTGRES_DB: magpie
+      POSTGRES_USER: magpie
+    restart: always
+
+  magpie:
+    image: pavics/magpie:latest
+    container_name: magpie
+    volumes:
+      - ./config/magpie/magpie.ini:/opt/local/src/magpie/config/magpie.ini
+      # - ./config/magpie/magpie.ini:/magpie.ini
+    environment:
+      FORWARDED_ALLOW_IPS: "*"
+      # MAGPIE_INI_FILE_PATH: /magpie.ini
```

Comment:
@fmigneault, did I missed something? I cannot get this to work.
Also, apart from overriding the magpie.ini I don't thing that we can override the default magpie.url using the environment variable MAGPIE_URL since the ini has precedence?

Reply:
Setting MAGPIE_CONF_DIR cause new issues, I guess that something else must be put in that directory